l10n_br_sale_product/models/sale_line.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# @ 2019 Akretion - www.akretion.com.br -
+#   Magno Costa <magno.costa@akretion.com.br>
```

Comment:
as this file is largely copied from sale.py, it should also mention
Copyright (C) 2014  Renato Lima - Akretion